src/pages/Atlas/index.tsx

```
@@ -0,0 +1,157 @@
+  maximumFractionDigits: 1,
+});
+const formatStake = (stake: number) => {
+  return STAKE_FORMAT.format(stake);
+}
+const renderStake = (ctx: CellContext<RowData, number>) => {
+  return <>{ formatStake(ctx.getValue()) }</>;
+}
+
+const columnHelper = createColumnHelper<RowData>();
+
+const queryDns = async (domain: string, recordType: string) : Promise<Array<unknown>> => {
```

ESLint

[error] 'queryDns' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)